sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -160,8 +188,25 @@ export class SenderImpl implements Sender {
     if (!Array.isArray(messages)) {
       messages = [messages];
     }
-    const sender = MessageSender.create(this._context);
-    return sender.sendBatch(messages);
+    return this._sender.sendBatch(messages);
+  }
+
+  async createBatch(options?: CreateBatchOptions): Promise<SendableMessageInfoBatch> {
+    this._throwIfSenderOrConnectionClosed();
+    if (!options) {
+      options = {};
+    }
```

Comment:
`this._sender.createBatch` does a similar check. So, we should be able to skip this

Reply:
Makes sense, removed!